label-studio/tests/test_pachyderm_storage.py

```diff
@@ -0,0 +1,237 @@
+import pytest
+import json
+
+from label_studio_sdk import Client as LSClient
+from pachyderm_sdk import Client as PachClient
+from pachyderm_sdk.api import pfs
+
+from tests.constants import LABEL_STUDIO_URL, AUTH_TOKEN, PACHD_ADDRESS
+
+
+@pytest.fixture(scope="session")
+def pach_client():
+    return PachClient.from_pachd_address(PACHD_ADDRESS)
+
+
+@pytest.fixture
+def pachyderm_data(pach_client):
+    import_repo = pfs.Repo(name="images", type="user")
+    pach_client.pfs.create_repo(repo=import_repo)
+
+    with pach_client.pfs.commit(
+        branch=pfs.Branch(repo=import_repo, name="master")
+    ) as commit:
+        commit.put_file_from_url(
+            path='/liberty.jpg',
+            url='https://docs.pachyderm.com/images/opencv/liberty.jpg',
+        )
+        commit_id = commit.id
+
+    export_repo = pfs.Repo(name="labels", type="user")
+    pach_client.pfs.create_repo(repo=export_repo)
+    branch = pfs.Branch(repo=export_repo, name="master")
+    pach_client.pfs.create_branch(branch=branch)
+
+    yield import_repo, commit_id, export_repo
+
+    pach_client.pfs.delete_all()
+
+
+@pytest.fixture(scope="session")
+def project():
+    ls_client = LSClient(url=LABEL_STUDIO_URL, api_key=AUTH_TOKEN)
+    ls_client.check_connection()
+
+    project = ls_client.start_project(
+        title="Image Annotation",
+        label_config="""
+    <View>
+        <Image name='image' value='$image'/>
+        <RectangleLabels name='label' toName='image'>
+            <Label value='Statue' background='green'/>
+        </RectangleLabels>
+    </View>
+    """,
+    )
+
+    return project
+
+
+@pytest.fixture
+def import_storage_id(pachyderm_data, project):
+    import_repo, commit_id, _ = pachyderm_data
+
+    # Create import storage
+    payload = {
+        "pach_repo": import_repo.name,
+        "pach_commit": commit_id,
+        "pachd_address": PACHD_ADDRESS,
+        "use_blob_urls": True,
+        "title": "Images",
+        "project": project.id,
+    }
+    response = project.make_request("POST", "/api/storages/pachyderm", json=payload)
```

Comment:
Is this the workaround for not updating the label studio client, or is this just how it works?

Reply:
This is just how it works. In the Label Studio SDK, the python methods for connecting to google storage, aws storage, etc. are just friendly wrappers around the same `project.make_request(METHOD, URL, json=payload)` call. [For example](https://github.com/HumanSignal/label-studio-sdk/blob/master/label_studio_sdk/project.py#L1292).